src/screens/Profile/Header/Shell.tsx

```diff
@@ -68,10 +76,22 @@ let ProfileHeaderShell = ({
           },
         ],
         index: 0,
-        thumbDims: null,
       })
+    },
+    [openLightbox],
+  )
+
+  const onPressAvi = React.useCallback(() => {
+    const modui = moderation.ui('avatar')
+    const avatar = profile.avatar
+    if (avatar && !(modui.blur && modui.noOverride)) {
+      runOnUI(() => {
+        'worklet'
```

Comment:
just a question. is this one of those cases now where it will complain if you don't add in `worklet`?

Reply:
I believe so; it's only automatically added for reanimated's hooks